elliottlib/util.py

```diff
@@ -238,6 +238,35 @@ def parallel_results_with_progress(inputs, func, file=None):
     return results
 
 
+def get_target_release(bugs):
```

Comment:
Looks like a good subject for unit or functional tests

Reply:
+1 will add it